src/pages/index.tsx

```diff
@@ -127,100 +127,115 @@ export default function Page({ modelIds, modelData }: Props) {
-                                                                <div className="cursor-pointer rounded-lg bg-accent-500 px-2 py-1 text-sm font-medium text-gray-100 transition-colors ease-in-out hover:bg-accent-600 hover:text-gray-100 dark:bg-accent-600 dark:text-gray-100 dark:hover:bg-accent-700">
-                                                                    {architecture}
-                                                                </div>
-                                                                <div className="cursor-pointer rounded-lg bg-accent-500 px-2 py-1 text-sm font-medium text-gray-100 transition-colors ease-in-out hover:bg-accent-600 hover:text-gray-100 dark:bg-accent-600 dark:text-gray-100 dark:hover:bg-accent-700">
-                                                                    {scale}x
+                                                    />
+
+                                                    <div className="relative inset-x-0 bottom-0 bg-white p-4   dark:bg-fade-900">
+                                                        <Link href={`/models/${id}`}>
+                                                            <div className="block text-2xl font-bold text-gray-800 dark:text-gray-100">
+                                                                {id}
+                                                            </div>
+                                                        </Link>
+                                                        <div className="text-gray-600 dark:text-gray-400">
+                                                            <div className="flex">
+                                                                <div className="mr-1">
+                                                                    {startsWithVowel(architecture) ? 'an' : 'a'}
                                                                 </div>
+                                                                <Link href={`/architectures/${architecture}`}>
+                                                                    <div className="mr-1 font-bold text-accent-500">
+                                                                        {architecture}
+                                                                    </div>
+                                                                </Link>
+                                                                <div className="mr-1">model by</div>
+                                                                {asArray(author).map((userId) => (
+                                                                    <React.Fragment key={userId}>
+                                                                        <Link href={`/users/${userId}`}>
+                                                                            <div className="font-bold text-accent-500">
+                                                                                {userId}
```

Comment:
Joey, enough is enough. You are 20 indentation levels deep. Please start extracting things out into functions.

Reply:
I got a little carried away I guess lmao